R/panels.R

```diff
@@ -76,7 +76,7 @@ panel <- function(head = NULL, body = NULL, footer = NULL,
           can_collapse
       ),
       div(class="panel-body", id = id_body,
-          div(class="panel-content",
+          div(class="panel-content",icon(""),
```

Comment:
@facarranza porque la funcion de incon sin argumento?, tiene sentido que entre como argumento otra opción que no sea ""?

Reply:
@CamilaAchuri EL problema surge por alguna incompatbilidad entre shinypanles y parmesan con respecto al tooltip, pueda ser porque ambos paquetes manejan tooltips de diferente manera,  este ejemplo de parmesan se ejecuta perfectamente https://github.com/datasketch/parmesan/blob/master/inst/examples/ex08-info-tooltips/app.R. Sin Icon, el tooltip no se muestra.

Al incluir la librería shinypanels en la app-simple no reconoce el tooltip  (Icon) (https://github.com/datasketch/app-simple-charts-orgs/issues/4). Al agregar icon("") se supera  la incompatibildad,

Reply:
@facarranza y al llamarlo en la aplicacion (en este caso de graficos simples) aparece el icono?

Reply:
@CamilaAchuri hasta donde he logrado indagar lo que permite es que aparezca el ícono que viene  indicado en el parmesan ej: 
![image](https://user-images.githubusercontent.com/58829002/215874485-876dda56-0759-4d9d-9a38-5ef1089ee281.png)

` <div class="panel-body" id="_body">
            <div class="panel-content">
              <i class="far fa-" role="presentation" aria-label=" icon"></i>
              <div>
                <div style="display: block;">
                  <div style="display: flex;">
                    <div id="select_var" class="shiny-html-output"></div>
                    <div class="tooltip" style=""><span></span><div class="tooltip-slot"><span class="tooltip-icon" style=""><i class="fas fa-circle-info" role="presentation" aria-label="circle-info icon"></i></span><div class="tooltip-content" style=""><span class="i18n table_tooltip">Table entries can be deleted, corrected, copied and pasted (using ctrl c and ctrl v).  If you edit a cell remember to press the enter key to save the changes.</span></div></div></div>
                  </div>
                  <div style="overflow:auto;">
                    <div id="dataset" class="shiny-html-output"></div>
                  </div>
                </div>
              </div>
            </div>
          </div>`